src/stirling/source_connectors/socket_tracer/protocols/dns/stitcher.cc

```diff
@@ -22,13 +22,24 @@
 #include <rapidjson/stringbuffer.h>
 #include <rapidjson/writer.h>
 
+#include <chrono>
 #include <deque>
 #include <string>
 #include <utility>
 
 #include "src/common/base/base.h"
 #include "src/stirling/source_connectors/socket_tracer/protocols/dns/types.h"
 
+DEFINE_bool(include_respless_dns_requests, false,
+            "If true, use customStitchFrames otherwise uses simple StitchFrames");
+
+DEFINE_uint64(dns_request_timeout_threshold_milliseconds, 2000,
+              "Number of seconds to wait for the in-flight response of a dns request.");
```

Comment:
Can you add, in the description, that this flag depends on `include_respless_dns_requests`?

Reply:
fixed in ef1516a28aab39fd18e928ea5eb157412e71716d